### PR TITLE
Use amazon cloudwatch agent

### PR DESCRIPTION
**Pull request by JoeARO**
This PR removes the bbc-statsd cloudwatch agent, configures the amazon cloudwatch agent to collect statsd metrics and adds a global BBCEnvironment tag to metrics emitted from ExMetrics.

### CODE REVIEW

File: rel/runtime_config.exs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+use Mix.Config
+
+[
+  "PRODUCTION_ENVIRONMENT"
```
Comment (ettomatic) [inline rel/runtime_config.exs:4]: I guess IDCTA will come at a later PR isn't it?
https://github.com/bbc/belfrage/blob/master/rel/runtime_config.exs#L25

**Review by ettomatic — APPROVED** [review]

File: rel/runtime_config.exs (hunk 1 above)
Comment (samfrench) [reply rel/runtime_config.exs:4]: Fetcher doesn't use IDCTA.

File: rel/runtime_config.exs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+]
+|> Enum.each(fn config_key ->
+  if System.get_env(config_key) == nil do
+    raise "Config not set in environment: #{config_key}"
+  end
+  config :belfrage_ccp,
+         Keyword.new([{String.to_atom(String.downcase(config_key)), System.get_env(config_key)}])
+end)
```
Comment (samfrench) [inline rel/runtime_config.exs:12]: Same question about this as asked on the other PR.

Comment (JoeARO) [reply rel/runtime_config.exs:12]: Its setting the `PRODUCTION_ENVIRONMENT` variable in the list, I just copied what we do there from belfrage/ccp

**Review by thymusvulgaris — APPROVED** [review]

**Review by samfrench — APPROVED** [review]

**Pull request merged**